drivers/bluetooth/hci/hci_b91.c

```diff
@@ -0,0 +1,304 @@
+/*
+ * Copyright (c) 2022, Telink Semiconductor (Shanghai) Co., Ltd.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#define BT_DBG_ENABLED IS_ENABLED(CONFIG_BT_DEBUG_HCI_DRIVER)
+#define LOG_MODULE_NAME bt_hci_driver_b91
+#include "common/log.h"
+
+#include <sys/byteorder.h>
+#include <drivers/bluetooth/hci_driver.h>
+
+#include <b91_bt.h>
+
+#define HCI_CMD                 0x01
+#define HCI_ACL                 0x02
+#define HCI_SCO                 0x03
+#define HCI_EVT                 0x04
+#define HCI_ISO                 0x05
+
+#define HCI_BT_B91_TIMEOUT K_MSEC(2000)
+
+static K_SEM_DEFINE(hci_send_sem, 1, 1);
+
+static bool is_hci_event_discardable(const uint8_t *evt_data)
```

Comment:
Not your fault, but we should probably factor out this kind of functions to a common helper, probably in `include/drivers/bluetooth/hci_driver.h`, since it doesn't really make sense to repeat the same code in every single driver. Anyway, this can be done as a later pull request, so no action needed here right now (except maybe the pavement of the verg "is", i.e. call it `hci_event_is_discardable()`

Reply:
Agree.